nvflare/private/fed/app/fl_app_validator.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import Dict, Tuple, Optional, List
+
+from nvflare.apis.app_validation import AppValidator, AppValidationKey
+
+
+class DefaultAppValidator(AppValidator):
```

Comment:
Maybe should change the file name to default_app_validator.py?

Reply:
add a new file called "default_app_validator.py" to put in this class.